concent_api/core/message_handlers.py

```diff
@@ -77,7 +77,7 @@ def handle_send_force_report_computed_task(client_message):
         task_to_compute,
     )
 
-    if Subtask.objects.filter(
+    if Subtask.objects.filter(  # pylint: disable=no-member
```

Comment:
On which version of pylint this problem occurs ?

Reply:
pylint==1.9.2
pylint-django==0.11.1